seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/client/HttpClientProvider.java

```diff
@@ -115,14 +120,47 @@ public HttpResponse execute(
             String method,
             Map<String, String> headers,
             Map<String, String> params,
-            String body)
+            Map<String, Object> body,
+            Map<String, Object> pageParams,
```

Comment:
as a http request until, why we need care pageParams? Can we handle it in https://github.com/apache/seatunnel/blob/b159cc0c75719a2ae73ec7e5a4bdc4bf63d4d7c8/seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/source/HttpSourceReader.java#L148?